File: Data Structures & Algorithms/reverse-string/submission-0.java
class Solution {
    public void reverseString(char[] s) {
        int i=0,j=s.length-1;
        while(i<j){
            char h=s[i];
            s[i]=s[j];
            s[j]=h;
            i++;
            j--;
        }
        System.out.println(s);
    }
}

File: Data Structures & Algorithms/find-lucky-integer-in-an-array/submission-0.py
class Solution:
    def findLucky(self, arr: List[int]) -> int:
        freq= {}
        for i in arr:
            if i in freq:
                freq[i]+=1
            else:
                freq[i]=1
        res=-1
        for key,value in freq.items():
            if key==value:
                res=max(res,key)

        return res